fsd_utils/mapping/application/multi_input.py

```diff
@@ -48,37 +49,43 @@ def formatted_values(value):
             )
 
         return (
-            f"{cls.indent}- {key}: {formatted_values(value)}"  # noqa
+            f"{cls.indent}. {key}: {formatted_values(value)}"  # noqa
             if index != 1
-            else (f"- {key}: {formatted_values(value)}")  # noqa
+            else (f". {key}: {formatted_values(value)}")  # noqa
         )
 
     @classmethod
     def format_nested_data(cls, value):
+        """
+        Formats nested data based on specific keys and data and returns the formatted result.
+        Args:
+            value: A nested data structure to be processed and formatted.
 
-        formatted_nested_values = []
-        for inner_items in value:
+        Returns:
+            str: The formatted result obtained from the nested data.
+        """
 
-            for k, v in inner_items.items():
-                for iso_keys in ["date", "month", "year"]:
-                    try:
+        formatted_nested_values = []
+        try:
+            for inner_items in value:
+                for k, v in inner_items.items():
+                    for iso_keys in ["date", "month", "year"]:
                         if iso_keys in k.split("__"):
-                            formatted_nested_values.append(f"{iso_keys}: {v}")
+                            v = number_to_month(v, iso_keys)
+                            formatted_nested_values.append(f"{v}")
                             break
 
                     # handles all other nested multiple values
-                    except:  # noqa
-                        formatted_nested_values.append(
-                            ", ".join(
-                                map(
-                                    lambda item: ", ".join(
-                                        [f"{k}: {v}" for k, v in item.items()]
-                                    ),
-                                    value,
-                                )
-                            )
-                        )
-        return formatted_nested_values
+        except:  # noqa
```

Comment:
Is the noqa comment meant to be here, have we not gotten QA on this section?

Reply:
No it means the linter should ignore that this is a catch-all exception block